tests/blackbox_test.go

```diff
@@ -347,3 +359,12 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 		return fmt.Errorf("Expected failure and ignition succeeded")
 	}
 }
+
+// Remove a LUKS device
+func removeLuksDevice(deviceName string) error {
+	cmd := exec.Command("sudo", "cryptsetup", "luksClose", deviceName)
```

Comment:
This should check first if the LUKS device exists (a simple check for `/dev/mapper/$name` should suffice).